lib/utils/report.mjs

```diff
@@ -276,7 +276,7 @@ function computeEditInfo({ fix, line, result: { stylelint }, ruleName }) {
 	// Compute the smallest range and text of the fix
 	const fixData = narrowFixRange(node, {
 		range: fixedNodeRange,
-		text: node.toString(),
+		text: node.parent?.raws.semicolon ? `${node.toString()};` : node.toString(),
```

Comment:
[suggestion] It seems better to move this change to another PR so that we can easily track the commit history later.

For example, PR #8367 is dedicated to one refactoring.

Any thoughts? @romainmenke

Reply:
Oh sorry, I missed #8370. We should merge it first. 😅